README.md

```diff
@@ -47,6 +47,23 @@ Credentials:
  password = 'XXX'
 ```
 
+### Docker
+
+You can use Docker to build an image with all the dependencies included.
+
+```
+# Build
```

Comment:
If you are keen to explore, there are automated builds on Docker Hub, which will link with the Github repository and automatically do the build process. 

The build step would then be removed form here, and the image name in the run steps (i.e. `wasg-register`) would have to be changed to `zerotypic/wasg-register`, assuming that your Docker Hub ID is `zerotypic`

Reply:
Thanks! I'll look at the Docker Hub stuff another day; I don't use it.